inventories/seapath_vm_definition_example.yml

```diff
@@ -17,6 +17,8 @@ all:
           bridges:
             - name: "br0" # Change the bridge name
               mac_address: "52:54:00:c4:ff:02" # change the MAC address
+              vlan:
+                vlan: 100 # Add a VLAN on the OVS port created
```

Comment:
br0 is not an OVS bridge (in your example it's "ovs0"), so maybe this example is misleading

Reply:
Fixed